src/ast/passes/codegen_llvm.h

```diff
@@ -259,7 +258,11 @@ class CodegenLLVM : public Visitor {
   int current_usdt_location_index_{ 0 };
   bool inside_subprog_ = false;
 
-  std::map<std::string, AllocaInst *> variables_;
+  struct VariableLLVM {
+    llvm::Value *value;
```

Comment:
It's because on this line we we'll have a LoadInst instead of an AllocaInst: https://github.com/bpftrace/bpftrace/pull/3014/files#diff-28926c090177e6efbfb5a7449833e0e863bb226822abb513b7dfc6c388837d87R4257

We're loading a pointer from the ctx tuple and using that to access our variable's memory, instead of going directly through the original AllocaInst.